src/metabase/pulse/parameters.clj

```diff
@@ -21,10 +23,13 @@
    (the-parameters subscription dashboard)))
 
 (defn value-string
-  "Returns the value of a dashboard filter as a comma-separated string"
+  "Returns the value(s) of a dashboard filter, formatted appropriately."
   [parameter]
-  (let [values (u/one-or-many (or (:value parameter) (:default parameter)))]
-    (str/join ", " values)))
+  (let [tyype  (:type parameter)
+        values (or (:value parameter) (:default parameter))]
+    (try (shared.params/formatted-value tyype (public-settings/site-locale) values)
```

Comment:
This is an extra change -- I figured we might as well use the param formatting code to also format the parameter values listed in the dashboard subscription header. Currently it just shows the raw value which isn't very pretty in some cases.